packages/openactive-integration-tests/test/types/OpportunityCriteria.d.ts

```diff
@@ -0,0 +1,9 @@
+export interface OpportunityCriteria {
+  opportunityType: string | null;
+  opportunityCriteria: string;
+  primary: boolean;
+  control: boolean;
+  opportunityReuseKey?: number;
+  usedInOrderItems?: number;
+  seller?: string;
+}
```

Comment:
Here I'm using a `.d.ts` file to specify a type within TypeScript itself. This type is now used across multiple files so it didn't make sense to localize it only in feature-helper